env-example

```diff
@@ -17,5 +17,9 @@ NEW_RELIC_APP_NAME=
 NEW_RELIC_LICENSE_KEY=
 GA_KEY=
 
+# dev server
+PORT=4202
```

Comment:
Note: you _MUST_ copy these to your actual `.env`, or things won't work.

The underlying `ng serve` doesn't know what actual HOST your browser is connecting to, as it's behind several proxies now!  And I think the default PORT=4200 will be used unless you have it in your ENV.